hyperliquid/exchange.py

```diff
@@ -78,14 +78,15 @@ def _slippage_price(
         is_buy: bool,
         slippage: float,
         px: Optional[float] = None,
+        is_spot: bool = False,
```

Comment:
You should be able to figure out if it is spot based on coin

Reply:
Good point, fixed!